app/views/groups/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <%= stylesheet_link_tag 'groups/index' %>
+</head>
+<body>
+<header class="fixed" >
+    <div>
+        <%= image_tag 'menu.png', id: "menu-icon" %>
+    </div>
+    <h2>
+        Categories
+    </h2>
+    <div>
+        <%= button_to "Logout", destroy_user_session_path, method: :delete %>
+    </div>
+</header>
+
+<div id="group">
+    <ul>
+        <% @groups.each do |group| %>
+            <%= link_to group_expenses_path(group_id: group.id) do %>
+                <li class="group-container">
+                    <div class="name-icon-container">
+                        <%= image_tag url_for(group.icon), alt: "group icon", class: "group-icon" %>
+                        <h3 class="no-decoration">
+                            <%= group.name %>
+                        </h3>
+                    </div>
+                    <h3 class="no-decoration">
+                        $<%= Expense.where(groups_id: group.id).sum(:amount) %>
+                    </h3>
+                </li>
+            <% end %>
+        <% end %>
+    </ul>
```

Comment:
- [ ] kindly make your font and text color similar to the given design guide, also remove the text underline on the group name and price
your design :point_down: 
![Screenshot from 2023-08-12 09-42-33](https://github.com/hasnatali1947/rails_capston/assets/66097371/1637492c-94ee-4de9-ad68-423bae42bf43)
given design :point_down: 
![Screenshot from 2023-04-27 19-57-40](https://github.com/hasnatali1947/rails_capston/assets/66097371/454ed82e-c684-403b-b128-eae9e371eed5)